demo

```diff
@@ -22,6 +22,10 @@ if [ "$#" -eq 0 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
     exit 1
 fi
 
+# For the integration tests the name of the docker-compose project is
+# generated by pytest
+DOCKER_COMPOSE_PROJECT_NAME="${DOCKER_COMPOSE_PROJECT_NAME:-integration}"
+
```

Comment:
One missing detail, though, will be to check if the user has passed a "-p something" in the arguments and remove it, else you will be passing it two times to docker-compose